Features/DeleteFeature

```diff
@@ -0,0 +1,10 @@
+@test
```

Comment:
Each of the feature files you have created contains the tag "@test."  You should consider a strategy around how tags will be used within the project. When you execute tests using the command line, you typically use tags to tell the system what tests to run. You can use tags at the feature, scenario, and scenario outline level.

For example here, the top-level tag(s) for this file might be:
@servicetests @deletetests

where @servicetests is added to each of the feature files you have added, and @deletetests is unique to this feature file.